components/DaemonProgress.qml

```diff
@@ -26,13 +26,13 @@
 // STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 // THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import QtQuick 2.0
+import QtQuick 2.6
```

Comment:
2.6 is available in QT 5.6 and above. Can @mbg033 confirm if QtQuick > 2.0 is an issue?